client/src/components/TaskLine.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import GlobalStyles from './GlobalStyles';
+import Dot from '../assets/dot.svg';
+import CategoryButton from './CategoryButton';
+
+const Task = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 5px 15px 5px 10px;
+`;
+
+const Time = styled.p`
+  font-size: 15px;
+`;
+
+const Container = styled.div`
+  width: 188px;
+  height: 30px;
+  font-size: 15px;
+  border: 3px solid #b9b9b9;
+  border-radius: 7px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+`;
+
+const TaskHeading = styled.h2`
+  font-size: 15px;
+  font-weight: 400;
+`;
+
+function TaskLine() {
+  return (
+    <>
+      <GlobalStyles />
```

Comment:
I think you don't need the global styles here, because you have implement the global styles in the app.js

Reply:
I need them here because if it s only in the app.js in only works when I build the pages and not for the components

Reply:
if you add GlobalStyles to you storybook `config.js` you can use them for all components that are rendered in storybook. Then later you can add them to `App.js` and have the exact same effect without having to add them to every component